src/tags/videotag.js

```diff
@@ -89,50 +104,69 @@ const VideoTag = class VideoTag extends HtmlTag {
   }
 
   content() {
-    var fallback, innerTags, type, sourceTransformation, sourceTypes, src, srcType, transformation, videoType;
-    sourceTypes = this.transformation().getValue('source_types');
-    sourceTransformation = this.transformation().getValue('source_transformation');
+    var fallback, sourceTypes, sources;
+    sources = this.transformation().getValue('sources');
     fallback = this.transformation().getValue('fallback_content');
-    if (isArray(sourceTypes)) {
-      let options = this.getOptions();
-      innerTags = (function () {
-        var i, len, results;
-        results = [];
-        for (i = 0, len = sourceTypes.length; i < len; i++) {
-          srcType = sourceTypes[i];
-          transformation = sourceTransformation[srcType] || {};
-          src = url(`${this.publicId}`, defaults({}, transformation, {
-            resource_type: 'video',
-            format: srcType
-          }, options));
-          videoType = srcType === 'ogv' ? 'ogg' : srcType;
-          type = 'video/' + videoType;
-          results.push(`<source ${this.htmlAttrs({src, type})}>`);
-        }
-        return results;
-      }).call(this);
-    } else {
-      innerTags = [];
-    }
+    sourceTypes = this.transformation().getValue('source_types');
+    var innerTags = sources ?
```

Comment:
Behavior in case of `sources` has an empty array value is different from other SDKs. E.g. PHP SDK explicitly tests for non empty array https://github.com/cloudinary/cloudinary_php/commit/7bcbcf3b43f345dddd991eac999530da312e47a7#diff-74609c4f68d4d6a89645762309671052R733